webui/react/src/pages/WorkspaceList/WorkspaceActionDropdown.tsx

```diff
@@ -122,13 +114,13 @@ export const useWorkspaceActionMenu: (props: WorkspaceMenuPropsIn) => WorkspaceM
       handlePinClick();
     },
     [MenuKey.Edit]: () => {
-      handleEditClick();
+      WorkspaceEditModal.open();
     },
     [MenuKey.SwitchArchived]: () => {
       handleArchiveClick();
     },
     [MenuKey.Delete]: () => {
-      handleDeleteClick();
+      WorkspaceDeleteModal.open();
     },
   };
```

Comment:
out of scope for this pr, but why not just map the keys to the functions directly?

Reply:
not aware of a good reason we have it wrapped. my guess is that this structure was just copied from places where it was necessary for passing arguments or calling multiple functions, even in places where that didn't apply.

Reply:
i think i originally started this before. the reason why its wrapped is that its more flexible for the future use and debugging (no need to wrap `{}` block if you wanna write multiple lines of code from one liner). Also diff tends to be concise. Its a bit longer but i believe its readable and scalable.

```diff
- const a = () => someFunc();
+ const a = () => {
+   someFunc();
+  someFunc2()
+ }
``` 

vs 


```diff
const a = () => {
   someFunc();
}
const a = () => {
   someFunc();
+  someFunc2()
}
```